core/data.go

```diff
@@ -326,15 +324,15 @@ type EncodedBundles map[QuorumID]*ChunksData
 
 // BlobMessage is the message that is sent to DA nodes. It contains the blob header and the associated chunk bundles.
 type BlobMessage struct {
-	BlobHeader *BlobHeader
```

Comment:
Do we need the concept of certificate internally? It looked nicer/more intuitive to have blob header + body (chunks/bundle).

Reply:
I think it's pretty useful to maintain the distinction between the two since they have different purposes. Here it makes sense that it is the certificate because we are asking the DA node to sign the certificate. 

That being said, I'm open to other naming suggestions!